pfcp/message/send.go

```diff
@@ -479,6 +490,7 @@ func SendPfcpSessionDeletionRequest(upNodeID pfcpType.NodeID, ctx *smf_context.S
 			}
 		}
 	} else {
+		InsertPfcpTxn(message.Header.SequenceNumber, &upNodeID)
```

Comment:
Can you check code - 	seqNum = getSeqNumber()
 Need to make sure sequence number is unique.

Reply:
looks good for single instance. but for multiple instance it might need some changes. we can take it up as a separate PR.

Reply:
Please open JIRA task for this